ostd/src/arch/loongarch/qemu.rs

```diff
@@ -15,7 +15,7 @@ pub enum QemuExitCode {
 
 /// Exits QEMU with the given exit code.
 //  FIXME: Support the transfer of the exit code to QEMU and multiple platforms.
-pub fn exit_qemu(_exit_code: QemuExitCode) -> ! {
+pub fn exit_qemu(_exit_code: QemuExitCode) {
```

Comment:
By changing the return type, the semantics of the function is changed. The previous form provides the guarantee that the kernel will exit. But the new form is saying that the action of QEMU exit may not succeed. To avoid confusion, I suppose that it makes sense to change the function name to `try_exit_qemu`. And you might want to extend the function's Rust doc to explain in what circumstances that this function would return.